prebid-server/developers/add-a-module.md

```diff
@@ -11,10 +11,6 @@ title: Prebid Server | Developers | Adding a Module
 This document guides you through the process of developing a module for host companies to plug into their instance of Prebid Server.
 We encourage you to look at existing modules for working examples. You can also ask us questions by [submitting a GitHub issue](https://github.com/prebid/prebid-server/issues/new).
 
-{: .alert.alert-info :}
-Modules are currently only supported in [PBS-Java](https://github.com/prebid/prebid-server-java).
-
-
```

Comment:
On line 22, can you also remove `(Java)` from `Prebid Server (Java) supports a rich module interface that`?